cql3/operation.hh

```diff
@@ -19,12 +20,12 @@
 
 #include <optional>
 
-namespace service::broadcast_tables {
-    class update_query;
-}
-
 namespace cql3 {
 
+namespace statements {
+    struct prepared_update_query;
```

Comment:
I would suggest a more specific name to make it clear that this is specific to broadcast tables / strongly consistent statements. Or namespacing.
e.g. `cql3::statements::broadcast_tables::prepared_update` or `cql3::statements::strongly_consistent::prepared_update` or something like that.

Reply:
Renamed to `cql3::statements::broadcast_tables::prepared_update`.